themes/stanford-style/lms/templates/register-shib.html

```diff
@@ -0,0 +1,193 @@
+<%inherit file="main.html" />
```

Comment:
Maybe we can avoid overriding `register-shib.html` if in case of standford ` marketing_link('HONOR') ` can be explicitly set.